webpack.config.js

```diff
@@ -0,0 +1,39 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: {
+    index: './src/index.js',
+  },
+  devtool: 'inline-source-map',
+  devServer: {
+    static: './dist',
+  },
+  // optimization: {
+  //   runtimeChunk: 'single',
+  // },
```

Comment:
- [ ] [OPTIONAL] Your configurations work well, but I think for best practice purposes, it's probably best to remove this commented piece of code to keep your work clean and professional. Leaving unused codes or comments with no spectacular reasons can often make your codebase messy or hamper understandability. In fact, it can increase your chance of bugs :thinking: which I know you don't want. Kindly remove this at your convenience, please. :pray: